ui/src/app/components/breadcrumb.tsx

```diff
@@ -179,7 +182,7 @@ export const Breadcrumb = fp.flow(
 
     trail(): Array<BreadcrumbData> {
       return getTrail(
-        this.props.routeConfigData.breadcrumb,
+        this.props.routeConfigData.breadcrumb || this.props.reactRouteData.breadcrumb,
```

Comment:
Confirming: does the `reactRouteData` store get cleared with an empty object when you navigate away from the React routes? Or will this potentially contain stale data from the most recent route? Probably won't be a problem here..

Reply:
This will contain stale data when navigating away. Once a react route is engaged again the data in the store will be completely replaced.